DEVELOPMENT.md

```diff
@@ -0,0 +1,82 @@
+# Development Guide
+
+> **Note**
+> This assumes that you have already followed the steps in the [README.md](./README.md) to set-up your OpenShift Clusters, OpenDataHub, ACM, OpenShift Pipelines, etc.
+
+## Useful Information
+
+### Models/Apps
+
+The GitHub repository is split into two main parts, Pipelines and ACM. Working on one part usually doesn’t involve working with the other, so you usually won’t have to set-up both dev environments.
```

Comment:
Can you elaborate/provide an example of what you'd like to see please?

Reply:
If the user (developer) will work with Pipelines, what will they be able to achieve? They will likely be able to take input model in S3 plus some deployment `Containerfile`s, and setup the pipelines so that upon manual (?) execution of the pipeline, the new container image with the model and runtime is built, tested, pushed to a container registry, and a PR filed to update the checksum of the image in some deployment YAML.

Reply:
Shouldn't that information be put in the pipelines README then? e.g.

## Overview

This repo contains three pipelines (listed in typical order of execution):

* `azureml-container-pipeline`:
	* Downloads the model from Git, S3, or Azure. Can also use a model located in a `PersistentVolume`.
	* Downloads the `Containerfile` used to build the image from Git.
	* Verifies the model files and `Containerfile` are accessible by the pipeline.
	* Creates an `ImageStream` for testing and local use.
	* Builds the image using `buildah`.

* `test-mlflow-image-pipeline`:
	* Creates a `Deployment` using the image generated by `azureml-container-pipeline`.
	* Creates a `Service` to allow network traffic into the `Deployment`.
	* Tests model inference using the provided data.
	* Uploads the image to Quay.

* `gitops-update-pipeline`:
	* Clones the Git repository containing the ACM application code.
	* Updates the image reference used by the app.
	* Makes a Pull Request with the changes.